Content.Server/Weapons/Reflect/ReflectSystem.cs

```diff
@@ -10,6 +11,8 @@ public override void Initialize()
         base.Initialize();
         SubscribeLocalEvent<ReflectComponent, EnergySwordActivatedEvent>(EnableReflect);
         SubscribeLocalEvent<ReflectComponent, EnergySwordDeactivatedEvent>(DisableReflect);
+        SubscribeLocalEvent<ReflectComponent, EnergyShieldActivatedEvent>(ShieldEnableReflect);
+        SubscribeLocalEvent<ReflectComponent, EnergyShieldDeactivatedEvent>(ShieldDisableReflect);
```

Comment:
```suggestion
```

I don't think these need to be here at all.

Reply:
Without it, the reflection doesn't work. I've already tried it. I had to add it.

Reply:
This is fine then.